msg/Arrays.msg

```diff
@@ -0,0 +1,34 @@
+# Arrays of different types
+bool[3] bool_values
+byte[3] byte_values
+char[3] char_values
+float32[3] float32_values
+float64[3] float64_values
+int8[3] int8_values
+uint8[3] uint8_values
+int16[3] int16_values
+uint16[3] uint16_values
+int32[3] int32_values
+uint32[3] uint32_values
+int64[3] int64_values
+uint64[3] uint64_values
+string[3] string_values
+BasicTypes[3] basic_types_values
+Constants[3] constants_values
+Defaults[3] defaults_values
+bool[3] bool_values_default [true, false, true]
+byte[3] byte_values_default [1, 0, 100]
+char[3] char_values_default [1, 0, 200]
+float32[3] float32_values_default [1.125, 0.0, -1.125]
+float64[3] float64_values_default [3.1415, 0.0, -3.1415]
+int8[3] int8_values_default [10, 0, -10]
+uint8[3] uint8_values_default [1, 0, 200]
+int16[3] int16_values_default [3000, 0, -3000]
+uint16[3] uint16_values_default [1, 0, 4000]
+int32[3] int32_values_default [500000, 0, -500000]
+uint32[3] uint32_values_default [1, 0, 6000000]
+int64[3] int64_values_default [70000000, 0, -70000000]
+uint64[3] uint64_values_default [1, 0, 800000000]
```

Comment:
For all integer values I suggest the following default values to ensure the documented valid range is covered (as it is the case in the current fixtures):
* signed: 0, *max*, *min*
* unsigned: 0 (*min*), 1, *max*

Same for the default values in the sequence messages.

Reply:
I'm having an issue with `int64` min value. I would expect that it can hold `-9223372036854775808`, but I am getting warnings "integer constant is so large that it is unsigned". Maybe my math is just off by one, but increasing the value by one squelches the warnings.

Reply:
See c7e5d87

Reply:
> I would expect that it can hold `-9223372036854775808`

I agree that this is the expected valid minimum value.

> I am getting warnings "integer constant is so large that it is unsigned".

How does the generated code look like for this?

Reply:
This is the line from the warning:

```c
   msg->int64_values_default.data[2] = -9223372036854775808;
```

And I can repro with gcc:

```c
int64_t foo = -9223372036854775808;
```

Reply:
To clarify, it is defined in the generated struct as:

```c
int64_t int64_value;
```

Reply:
I would expect the value to have two trailing `l`s: like this https://github.com/ros2/rosidl/blob/d1edcaf35534e8860730370a62f1d6a024ee0f66/rosidl_generator_cpp/rosidl_generator_cpp/__init__.py#L193

Reply:
Maybe it should be `u`, see the example at the bottom of [integer literal](https://en.cppreference.com/w/cpp/language/integer_literal).

Reply:
I've updated this PR to have the expected value.

Reply:
Proposed fix: https://github.com/ros2/rosidl/pull/371